lms/djangoapps/course_structure_api/v0/urls.py

```diff
@@ -18,7 +18,7 @@
         r'^grading_policies/{}/$'.format(COURSE_ID_PATTERN),
         views.CourseGradingPolicy.as_view(),
         name='grading_policy'
-    ),
+    )
```

Comment:
Don't make changes like this unless you are fixing a quality issue in the automated tests.  (Note that it was also inconsistent with the other one in this file.)